src/main/kotlin/no/skatteetaten/aurora/boober/controller/v1/AuroraDeploymentSpecControllerV1.kt

```diff
@@ -54,19 +54,20 @@ class AuroraDeploymentSpecControllerV1(
         return Response(items = specs.map { renderSpecAsJson(it, includeDefaults) })
     }
 
+    // TODO: Når vi skal bruke gobo må vi returnere warnings her.
     @GetMapping("/{environment}/{application}")
     fun get(
         @PathVariable auroraConfigName: String,
         @PathVariable environment: String,
         @PathVariable application: String,
         @RequestParam(name = "overrides", required = false) overrides: String?,
-        @RequestParam(name = "includeDefaults", required = false, defaultValue = "true") includeDefaults: Boolean
+        @RequestParam(name = "includeDefaults", required = false, defaultValue = "true") includeDefaults: Boolean,
+        @RequestParam(name = "errorsAsWarnings", required = false, defaultValue = "false") errorsAsWarnings: Boolean
```

Comment:
Bør kanskje oppdatere noen controller tester til å query parameteren? 
er ikke store endringen på controlleren, men kanskje verdt det

Reply:
Vi mocker uansett svaret fra facaden og når vi skal over til å bruke gobo skal alt dette muligens endres på uansett så jeg ser ikke helt behovet for det.